examples/package.json

```diff
@@ -34,6 +34,7 @@
     "node": ">=18"
   },
   "dependencies": {
+    "@golem-sdk/pino-logger": "^1.0.1",
```

Comment:
this is not used in the examples yet, switching to an external package requires rewrite all examples